crates/rustc_codegen_spirv/src/symbols.rs

```diff
@@ -430,7 +432,9 @@ pub(crate) fn parse_attrs_for_checking<'a>(
             .into_iter()
             .chain(args.into_iter().map(move |ref arg| {
                 let span = arg.span();
-                let parsed_attr = if arg.has_name(sym.image_type) {
+                let parsed_attr = if arg.has_name(sym.generic_image_type) {
+                    SpirvAttribute::IntrinsicType(IntrinsicType::GenericImageType)
```

Comment:
Is there a reason this isn't in the `custom_attributes` array? Seems like that's the proper place for it.

Reply:
Well it's not how the `Image` was implemented, and this was just adapting from how that was implemented.

Reply:
That's because `image` has arguments, `generic_image` does not - `symbols.rs` is organized by syntax, not by semantics.